<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RowEditDialog</class>
 <widget class="QWidget" name="RowEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="createRows">
     <property name="title">
      <string>Edit Rows</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <item row="2" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>7</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="6">
       <widget class="QListWidget" name="rowList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="updateRow">
        <property name="text">
         <string>Update Row</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Row Text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="rowView">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QToolButton" name="addRow">
        <property name="text">
         <string>+</string>
        </property>
        <property name="icon">
         <iconset resource="../crochet.qrc">
          <normaloff>:/images/plus.svg</normaloff>:/images/plus.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="removeRow">
        <property name="text">
         <string>-</string>
        </property>
        <property name="icon">
         <iconset resource="../crochet.qrc">
          <normaloff>:/images/minus.svg</normaloff>:/images/minus.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QToolButton" name="moveUp">
        <property name="text">
         <string>/\</string>
        </property>
        <property name="icon">
         <iconset resource="../crochet.qrc">
          <normaloff>:/images/arrow_up.svg</normaloff>:/images/arrow_up.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QToolButton" name="moveDown">
        <property name="text">
         <string>\/</string>
        </property>
        <property name="icon">
         <iconset resource="../crochet.qrc">
          <normaloff>:/images/arrow_down.svg</normaloff>:/images/arrow_down.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../crochet.qrc"/>
 </resources>
 <connections/>
</ui>
